pages/desktop/details.py

```diff
@@ -551,10 +552,9 @@ class ContributionSnippet(Page):
 
         def __init__(self, base_url, selenium):
             Page.__init__(self, base_url, selenium)
-
-            WebDriverWait(self.selenium, self.timeout).until(
-                lambda s: s.find_element(*self._make_contribution_button_locator).is_displayed(),
-                "Timeout waiting for 'make contribution' button.")
+            self.wait.until(expected.visibility_of_element_located(
```

Comment:
Yes, my intention was to address the implicit wait behind the explicit wait, but essentially my new code does the same thing. I'll address removing implicit waits separately (I have a patch locally, but it needs more testing).